Jenkinsfile

```diff
@@ -82,7 +79,7 @@ def job = {
         config.docker_tag = config.ksql_db_version
     } else {
         // For non-release builds, we append the build number to the maven artifacts and docker image tag
-        config.ksql_db_artifact_version = config.ksql_db_version + '-' + env.BUILD_NUMBER
+        config.ksql_db_artifact_version = config.ksql_db_version.tokenize("-")[0] + '-' + env.BUILD_NUMBER
```

Comment:
we won't ever be doing snapshot builds anymore correct? So if you just remove the -SNAPSHOT from the ksql_db_version variable in the config you don't need the tokenize call here or line 77 anymore. would you still need this new variable config.ksql_db_artifact_version?

Reply:
Great point. An added benefit of this is that the build param `KSQLDB_VERSION` and the config var `ksql_db_version` are now of the same form, removing some existing confusion around the two. 

Made the update, though I left `docker_tag` as a separate var from `ksql_db_artifact_version` even though they're always the same since I think subsequent lines in this script are easier to read that way.